packages/build/tests/edge_handlers/tests.mjs

```diff
@@ -31,11 +31,12 @@ const getEdgeHandlerBundlePath = async function ({ outputDir, manifestPath }) {
   return `${outputDir}/${sha}`
 }
 
-const requireEdgeHandleBundle = function (bundlePath) {
+const requireEdgeHandleBundle = async function (bundlePath) {
   const set = spy()
   global.netlifyRegistry = { set }
-  // eslint-disable-next-line node/global-require, import/no-dynamic-require
-  require(bundlePath)
+  const bundlePathWithExt = `${bundlePath}.js`
+  await fs.rename(bundlePath, bundlePathWithExt)
+  await import(bundlePathWithExt)
```

Comment:
The bundle created by the Edge handlers plugin does not have any extension even though it is JavaScript.
Unlike `require()`, `import()` needs a file extension.